src/composer/src/main.rs

```diff
@@ -67,10 +68,26 @@ pub fn exec() -> Result<(), String> {
         .rev()
         .collect();
     for c_id in reverse_ids.iter() {
+        
         sys.add_params_iter(&c_id, Parameters::transition_iter(c_id, &sys, &mut build)?);
+        let obj = ElfObject::transition_iter(c_id, &sys, &mut build)?;
         sys.add_objs_iter(&c_id, ElfObject::transition_iter(c_id, &sys, &mut build)?);
         sys.add_invs_iter(&c_id, Invocations::transition_iter(c_id, &sys, &mut build)?);
+        println!("path:{}", obj.get_path());
```

Comment:
Unless a print is for a specific purpose (other than debugging), we don't want to add them.

Reply:
Oh sorry. It is a debugging mistake.